package.json

```
@@ -15,22 +15,26 @@
     "url": "https://github.com/SonarSource/sonar-scanner-npm/issues"
   },
   "license": "LGPL-3.0-only",
-  "main": "src/index.js",
+  "main": "src/index.ts",
```

I think this should be
```suggestion
  "main": "build/index.js",
```

---

yup, I'll tweak this in the cli scanner work I'm doing now